src/ui/screen_saver.c

```diff
@@ -49,7 +50,7 @@ void screen_saver_process(void)
 {
     if (!_is_active) {
         _counter++;
```

Comment:
the counter should probably not be incremented either while disabled, for the same reason as in my other comment